reinforcement_learning/rl_game_server_autopilot/game-env/specs/autopilot-client.yaml

```diff
@@ -0,0 +1,105 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: autopilot
+  namespace: default
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - "*"
+  verbs:
+  - "*"
+- apiGroups:
+  - rbac.authorization.k8s.io
+  - extensions
+  - apps
+  resources:
+  - "*"
+  verbs:
+  - "*"
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  - pods
+  - deployments
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - delete
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: autopilot
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: autopilot
+  namespace: default
+subjects:
+- kind: ServiceAccount
+  name: autopilot
+  namespace: default
+roleRef:
+  kind: ClusterRole
+  name: autopilot
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  labels:
+    app: autopilot
+  name: autopilot
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: autopilot
+  template:
+    metadata:
+      labels:
+        app: autopilot
+    spec:
+      serviceAccountName: autopilot
+      containers:
+      - name: autopilot
+        image: 356566070122.dkr.ecr.us-west-2.amazonaws.com/autopilot:latest
```

Comment:
Hardcoded image. Is this a public image?

Reply:
It is public